t/t0012-content-sqlite.t

```diff
@@ -14,14 +14,14 @@ RPC=${FLUX_BUILD_DIR}/t/request/rpc
 
 HASHFUN=`flux getattr content.hash`
 
+test_expect_success 'load heartbeat module with fast rate to drive purge' '
+	flux module load heartbeat period=1s
```

Comment:
skimming the heartbeat module, could we make the period < 1s?  1s seems long in the unit tests (especially w/ the sleep 1 below).

Reply:
The sync callback rate is forced to be within `sync_min=1` and  `sync_max=10` seconds, so it doesn't help to make the heartbeat < 1s.  However, that min value was set to avoid triggering the heavyweight cache purge too frequently, and now with the LRU, the purge doesn't need to scan for eligible items and should exit immediately if there is nothing to do.  So maybe we can just eliminate the min value and crank the heartbeat down as you say.  I'll go ahead and do that.